contracts/context-config/src/lib.rs

```diff
@@ -17,12 +16,15 @@ mod sys;
 use guard::Guard;
 
 const DEFAULT_VALIDITY_THRESHOLD_MS: Timestamp = 10_000;
+const DEFAULT_CONTRACT: &[u8] = include_bytes!("../../proxy-lib/res/proxy_lib.wasm");
```

Comment:
embedding the proxy in this will only double the size, let's add a system method for updating it in the context config state